universal-application-tool-0.0.1/build.sbt

```diff
@@ -34,6 +34,9 @@ lazy val root = (project in file("."))
       "org.awaitility" % "awaitility" % "4.0.1" % Test,
       "org.mockito" % "mockito-core" % "3.1.0" % Test,
 
+      // General testing libraries
```

Comment:
suggestion (non-blocking): Consider linking to the testing library documentation https://javadoc.io/doc/com.google.guava/guava-testlib/latest/index.html

Reply:
Good idea! Done